napari/_vispy/vendored/volume.py

```diff
@@ -582,13 +583,24 @@
                     color = calculateColor(color, iloc, dstep);
                     gl_FragColor = applyColormap(color.r);
                     iter = nsteps;
+                    
+                    //Set depth value
+                    float clip_distance = length(farpos - nearpos);
+                    vec3 surface_point = iloc * u_shape;
```

Comment:
we should standardise iloc across the shaders/modes - maybe with a more useful name

Reply:
Agreed that we need some unification. The iso shader mode uses `iloc` because it performs the refined search for the surface in the main loop rather than in the `after_loop` like MIP, MINIP, etc. do.